torch/_higher_order_ops/flex_attention.py

```diff
@@ -71,6 +71,9 @@ def __call__(
             mask_mod_other_buffers,
         )
 
+    def pure(self):
+        return True
```

Comment:
@Chillee IIRC you had objections to the "pure" name, did you have other suggestions?